tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py27, py35, py36, flake8
 deps=
   pytest
   pytest-cov
+  cryptography
```

Comment:
Let's drop `cryptography`. GitHub Actions should handle this better than Travis CI.